### DISCUSSION TITLE
Ability to sync watch progress for multiple Jellyfin users

### DISCUSSION THREAD

**Discussion by botagas** [vosmiic/jellyfin-ani-sync#157] [Ideas]
I am not sure how difficult that would be or if it's possible, but recently I faced a dilemma where I wanted to have several users who would be able to sync their watch list with a provider. I do have Ani-Sync set up on my Jellyfin instance, but not a single plugin I searched has the ability to sync each user's watch progress with providers.

**Comment by vosmiic** [comment]
You are able to choose which user to update the provider/update your local Jellyfin progress on the manual sync tab. Or do you mean you would like to share the update after you complete an episode?

**Reply by botagas** [reply]
> You are able to choose which user to update the provider/update your local Jellyfin progress on the manual sync tab. Or do you mean you would like to share the update after you complete an episode?

No, I meant multi-user support, providing the ability for any Jellyfin user to sync their own watch progress to their respective MAL/AniList/etc. accounts.

**Reply by Samega7Cattac** [reply]
@vosmiic Basically setting a provider account for each jellyfin user.

**Reply by vosmiic** [reply]
I might still be misunderstanding but this exists. If you authenticate a user with a provider, and they login (the user that has been authenticated with a provider) and watch something, it will update their account watch progress. 
That is why an authentication link is generated, it is supposed to be sent to other users (assuming they are at a different computer) so they can authenticate with their own accounts.

**Reply by Samega7Cattac** [reply]
oh, so that's the workflow
Makes sense now
